src/Package/Impl/ProjectSystem/ProjectSettings.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.Common.Core;
+
+namespace Microsoft.VisualStudio.R.Package.ProjectSystem {
+    internal static class ProjectSettings {
+        public const string SettingsFilePattern = "*settings*.r";
```

Comment:
It seems that the pattern doesn't quite match the method below (it would match e.g. xsettingsy.r, but the method will return false).